### ISSUE TITLE
addon-a11y: When stories only contain disabled buttons, the color-contrast rule isn't triggered.

### ISSUE THREAD

**Issue by withinsight** [storybookjs/storybook#8088]
**Describe the bug**
When you have a story with only disabled buttons, the color-contrast rule isn't triggered.
I'm noticing color-contrast is running properly on all other stories. Only not working on the story with 100% disabled buttons.

**To Reproduce**
1. Create a story with only disabled buttons.
2. Observe the a11y addon not recognizing violations with buttons that clearly contain violations

**Expected behavior**
a11y addon would run the color-contrast rule and trigger a violation.

**Screenshots**
![image](https://user-images.githubusercontent.com/740056/64961262-0e03c200-d863-11e9-8fc0-004e917dc14a.png)

**System:**
Using storybook 5.2.0-rc.0

**Comment by withinsight** [comment]
@CodeByAlex here's the issue you asked me to tag you in.

**Comment by kgeorgiou** [comment]
I was looking to take a stab at this, but it seems axe-core **intentionally** ignores (some, if not all) disabled elements, including the `button`.
- [color-contrast-matches.js#L33-L38](https://github.com/dequelabs/axe-core/blob/c39c3a3db4975fb39365acace3fe1ffa7893004d/lib/rules/color-contrast-matches.js#L33-L38)
- [[test] color-contrast-matches.js#L195-L200](https://github.com/dequelabs/axe-core/blob/c39c3a3db4975fb39365acace3fe1ffa7893004d/test/rule-matches/color-contrast-matches.js#L195-L200).

**Comment by withinsight** [comment]
Thanks @kgeorgiou! Wow, is the assumption that color challenged users don't need to know when a button is disabled? That seems a bit foot in mouth.
